batch/file-ingest/pom.xml

```diff
@@ -8,12 +8,12 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.0.BUILD-SNAPSHOT</version>
+        <version>2.1.3.RELEASE</version>
```

Comment:
Boot is currently at 2.1.5, so I take we don't have a comparable parent and other dependencies that work nice together?

Reply:
2.1.5 works